### ISSUE TITLE
Freezes ST3 when editing large TSV files

### ISSUE THREAD

**Issue by invizibility** [SublimeText/ElasticTabstops#9]
i can edit just fine when the plugin is disabled.

**Comment by adzenith** [comment]
Yeah, it takes forever when the files are large. I tried to do some profiling using python's profile library, but it didn't reveal any obvious bottlenecks. I'm not sure what takes so long on large files.

**Comment by eugenesvk** [comment]
FYI I've added a PR https://github.com/SublimeText/ElasticTabstops/pull/19 that might offer some relief — the freezing happens only after some delay after the last edit, so at least it's not happening while you're actively typing.
The async API alternative mentioned in that PR would be the proper solution, but it bugs a bit
